packages/common/utils/team.js

```diff
@@ -0,0 +1,7 @@
+import { TEAM_TYPE } from '@caesar/common/constants';
+import { getDomainName } from './getDomainName';
+
+export const  getTeamTitle = ({ type, title }) =>
+  type === TEAM_TYPE.DEFAULT || title.toLowerCase() === TEAM_TYPE.DEFAULT
```

Comment:
I think it can confuse the programmer. I see, not an every team has a type. Why some team has type, bun some not include it? In my opinion you need to create type for every team.

Reply:
I've just checked it and found out that each team has type key. It may be default, other or null. 
I guess: check just type is enough.
UPD: I found where type is null. There is missed on backend. I asked @aburov-4xxi to add it.

Reply:
The thing is that there were both checks in other components so I decided to combine checks for type and title